addons/website/static/src/js/menu/seo.js

```diff
@@ -221,7 +221,7 @@ var HtmlPage = Class.extend(mixins.PropertiesMixin, {
         mixins.PropertiesMixin.init.call(this);
         this.initTitle = this.title();
         this.defaultTitle = $('meta[name="default_title"]').attr('content');
-        this.defaultDescription = $('meta[name="default_description"]').attr('content') || _t("The description will be generated by social media based on page content unless you specify one.");
+        this.defaultDescription = $('meta[name="default_description"]').attr('content') || _t("The description will be generated by search engines based on page content unless you specify one.");
```

Comment:
The top preview if for search engines, the bottom preview is for social media.

The text has to be adapted only on the top preview, not both.

So it needs to be split. A bigger refactoring here is probably needed, especially since this text is copy/pasted in 4 places! (of which you changed only 2)